src/main.ts

```diff
@@ -258,10 +258,10 @@ const createWindow = async () => {
 
 app.whenReady().then(
   async () => {
-    await createSplashWindow()
     await initSettings()
     appSettings = await getSettings()
     nativeTheme.themeSource = appSettings.appearance.theme
+    await createSplashWindow()
```

Comment:
If the OS settings don't match whatever is saved in the k6 Studio settings, there's a noticeable flash that occurs once the settings are loaded.